src/components/ux-subnavtitle/ux-subnavtitle.hbs

```diff
@@ -0,0 +1 @@
+<dt>{{>content}}:</dt>
```

Comment:
This subcomponent isn't called or used anywhere. I assume it's meant to be used if the subnav data has a title property?